oci/table_oci_objectstorage_object.go

```diff
@@ -0,0 +1,283 @@
+		Name:        "oci_objectstorage_object",
+		Description: "OCI Object Storage Object",
+		// Object can have same name in two different buckets, regions or compartments, leading to duplicate result in get call
+		List: &plugin.ListConfig{
+			Hydrate:       listObjectStorageObjects,
+			ParentHydrate: listObjectStorageBuckets,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the object.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "bucket_name",
+				Description: "The name of the bucket.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "namespace",
+				Description: "The Object Storage namespace used for the request.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "archival_state",
+				Description: "Archival state of an object. This field is set only for objects in Archive tier.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "cache_control",
+				Description: "The Cache-Control header.",
+				Hydrate:     getObjectStorageObject,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "content_disposition",
+				Description: "The Content-Disposition header.",
+				Hydrate:     getObjectStorageObject,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "content_encoding",
+				Description: "The Content-Encoding header.",
+				Hydrate:     getObjectStorageObject,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "content_language",
+				Description: "The Content-Language header.",
+				Hydrate:     getObjectStorageObject,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "content_range",
+				Description: "Content-Range header for range requests.",
+				Hydrate:     getObjectStorageObject,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "content_type",
+				Description: "The Content-Type header.",
+				Hydrate:     getObjectStorageObject,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "etag",
+				Description: "The current entity tag (ETag) for the object.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "expires",
+				Description: "The date and time after which the object is no longer cached by a browser, proxy, or other caching entity.",
+				Hydrate:     getObjectStorageObject,
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("Expires.Time"),
+			},
+			{
+				Name:        "is_not_modified",
+				Description: "Flag to indicate whether or not the object was modified.  If this is true,the getter for the object itself will return null.",
+				Hydrate:     getObjectStorageObject,
+				Type:        proto.ColumnType_BOOL,
+				Default:     false,
+			},
+			{
+				Name:        "md5",
+				Description: "Base64-encoded MD5 hash of the object data.",
```

Comment:
Does the `md5` column include sensitive data?

Reply:
This does not contain any sensitive data.